### ISSUE TITLE
Feature request: add a path option to the genius.save_lyrics()-function

### ISSUE THREAD

**Issue by ghwiegand** [johnwmillr/LyricsGenius#251]
From my view, there is no option to give the `save_lyrics()`-function a `path` or `destination`, where to save. I think that should be added. 
If I missed something or someone has a smart workaround, please let me know :)

**Comment by allerter** [comment]
You're right. `save_lyrics()` has a `filename` parameter which actually should be a `path` or `destination` parameter to determine the path and the filename altogether.  A PR doing this would be welcome.
One thing that's missing either way is that `save_lyrics` should return the JSON/TXT string if no filename/path is supplied (the method docs point this out, but it's not implemented).

**Comment by sufyan-siddiqui** [comment]
I think there should be a separate parameter for for the path as `path` or `destination` instead of using the `filename` parameter, since we are already giving a filename to file if not specified by the user.
